### PR TITLE
New Rule: Wise impersonation with suspicious attachment

### PR DESCRIPTION
**Pull request by aidenmitchell**

### CODE REVIEW

**Comment by aidenmitchell** [comment]
Running a hunt now, `*wise*` might be too broad.

**Comment by morriscode** [comment]
Edited rule to use coalesce, queuing  test rules again

**Review by morriscode — APPROVED** [review]
Rule has been in testing for a while with no FP's or TP's. Should be safe to deploy.

File: detection-rules/impersonation_wise_suspicious_attachment.yml [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+  Impersonating Wise Financial, as confirmed by NLU, and contains a suspicious attachment type.
+type: "rule"
+severity: "high"
+source: |
+  type.inbound
+  and (
+          strings.ilike(sender.display_name, '*wise*')
+          or strings.ilike(sender.email.domain.domain, '*wise*')
+      )
+  and any(ml.nlu_classifier(coalesce(body.html.display_text, body.plain.raw)).tags, .name in ("payment", "invoice"))
```
Comment (jkamdjou) [inline detection-rules/impersonation_wise_suspicious_attachment.yml:12]: @morriscode  we can update this to current thread

File: detection-rules/impersonation_wise_suspicious_attachment.yml [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+  and any(attachments, .file_type 
+      in~ $file_extensions_common_archives
+      or .file_type in~ $file_extensions_executables
+      or .file_type in~ $file_extensions_macros
```
Comment (jkamdjou) [inline detection-rules/impersonation_wise_suspicious_attachment.yml:16]: file type is already guaranteed to be lowercase

File: detection-rules/impersonation_wise_suspicious_attachment.yml (hunk 1 above)
Comment (morriscode) [reply detection-rules/impersonation_wise_suspicious_attachment.yml:12]: ahh for sure!

**Pull request merged**